src/assessor_tools.yml

```
@@ -242,6 +319,31 @@
         powershell: yes
         unarchive_extra_opts:
           - --strip-components=1
+    - role: assessment_tool
+      vars:
+        archive_src: https://github.com/lgandx/Responder/tarball/master
+        install_dir: /tools/Responder
+        pip_packages:
+          - crypto
+          - dnspython
+          - impacket
+          - ldap3
+          - readline
```

I think these packages are also required for this tool. `pycryptodome` is a drop-in replacement for `pycrypto` and uses the `Crypto` namespace. `pycryptodomex` is a library implementation of the same code that lives under the `Cryptodome` namespace. I believe both of them are in use.

Note: I used [pipreqs](https://github.com/bndr/pipreqs) to gather information about requirements.

```suggestion
          - pyasn1
          - pycryptodome
          - pycryptodomex
          - readline
          - six
```